src/app/components/account/account-list-item-layout.tsx

```diff
@@ -94,25 +92,19 @@ export function AccountListItemLayout(props: AccountListItemLayoutProps) {
               {isBitcoinEnabled && (
                 <Flex>
                   <Caption>{truncateMiddle(btcAddress, 5)}</Caption>
-                  {onCopyBtcToClipboard && (
-                    <Tooltip
-                      placement="right"
-                      label={hasCopiedBtc ? 'Copied!' : 'Copy Bitcoin address'}
-                      hideOnClick={false}
-                    >
-                      <Stack>
-                        <Box
-                          _hover={{ cursor: 'pointer' }}
-                          onClick={e => onCopyBtcToClipboard?.(e)}
-                          size="12px"
-                          color={color('text-caption')}
-                          data-testid={UserAreaSelectors.AccountCopyAddress}
-                          as={FiCopy}
-                          mt="2px"
-                          ml="4px"
-                        />
-                      </Stack>
-                    </Tooltip>
+                  {onClickBtcCopyIcon && (
+                    <Stack>
+                      <Box
+                        _hover={{ cursor: 'pointer' }}
+                        onClick={e => onClickBtcCopyIcon?.(e)}
+                        size="12px"
+                        color={color('text-caption')}
+                        data-testid={UserAreaSelectors.AccountCopyAddress}
+                        as={FiCopy}
```

Comment:
Let's put the FiCopy inside the box, so that this can be as="button" (otherwise it's bad accessibility)